ContactDetailsApi/V1/Domain/InvalidQueryParameterException.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ContactDetailsApi.V1.Domain
+{
+    public class InvalidQueryParameterException : Exception
```

Comment:
Is this exception class used anywhere?

Reply:
I thought i would use it, but I didn't, have deleted it